mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,143 @@
+"""Like Transporter but without Jason Statham."""
+from sys import stdout
+
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import GameSerializerForImportedData, PlayerScoreSerializer
+
+FIELD_REMAPPING = {
+    "game": {"map": "Default"},
+    "player_score": {
+        "corporation": {
+            "Interpl. Cinematics": "Interplanetary Cinematics",
+            "UNMI": "United Nations Mars Initiative",
+        }
+    },
+}
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    0: {"id": int},
+    1: {"date": parser.parse},  # datetime
+    2: {"game_map": str},
+    3: {"prelude": str_to_bool},
```

Comment:
You can use `bool` directly here

Reply:
![image](https://user-images.githubusercontent.com/34490649/93188406-d743de80-f738-11ea-9f6e-0e0de9f4e8c6.png)

Reply:
Oh shit I forgot it's 0 and 1 in the spreadsheet - thought you're checking for an empty string.